### PR TITLE
Further rebranding changes

### PR DESCRIPTION
**Pull request by marklewin**
## Description

The following regex turned up some outstanding Nexmo references in ADP ... changing them to Vonage.

```Nexmo(?!\s+CLI|\s+command|\sCall\sControl|\sOAS|[a-zA-Z]|\/|\\|\.[a-z]|::)```

(**Note:** You can [play with this regex here](https://regex101.com/r/HcdrV7/1/))

Files to exclude: ```*.html, *.json, .gitmodules, *.yml, *.php, *.py, *.rb, *.sh, /.repos, **/cn, **/ja```

Also changed references to Nexmo Developer to API Developer Portal (and NDP to ADP)

Per [DEVX-4776](https://nexmoinc.atlassian.net/browse/DEVX-4776)

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
 # Contributing
```
Comment (tbedford) [inline .github/CONTRIBUTING.md:1]: Probably shouldn't be touching the files in `.github`

File: README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
-# Nexmo Developer
+# API Developer Portal
```
Comment (tbedford) [inline README.md:1]: Should this be Vonage API Developer Portal?

File: README.md [hunk 3]
```diff
@@ -120,7 +120,7 @@ To stop the server press `ctrl+c`.
 
 You can access the admin dashboard by visiting `/admin`. Initially, you will have an admin user with the username of `api.admin@vonage.com` and password of `development`.
 
-The following is an example if you are running Nexmo Developer within a Docker container:
+The following is an example if you are running the API Developer Portal within a Docker container:
```
Comment (tbedford) [inline README.md:123]: Vonage API Developer Portal (and throughout this PR - I won't comment on all of them).

File: _extend/azure-speech-to-text.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ published: true
 
 
 ## Usage
-This code shows how to send audio from the [Nexmo Voice API Websocket](https://developer.nexmo.com/voice/voice-api/guides/websockets) to  [Azure Speech-to-text](https://azure.microsoft.com/en-gb/services/cognitive-services/speech-to-text/), it allows you obtain real time transcription of the callers speech.
+This code shows how to send audio from the [Voinage Voice API Websocket](https://developer.nexmo.com/voice/voice-api/guides/websockets) to  [Azure Speech-to-text](https://azure.microsoft.com/en-gb/services/cognitive-services/speech-to-text/), it allows you obtain real time transcription of the callers speech.
```
Comment (tbedford) [inline _extend/azure-speech-to-text.md:13]: Typo in Vonage.

File: _use_cases/en/number-insight-async-tutorial.md [hunk 5]
```diff
@@ -53,17 +53,17 @@ $ npm install express body-parser  --save
...
+When the Number Insight API finishes processing your request, it alerts your application via a [webhook](/concepts/guides/webhooks). The webhook provides a mechanism for Vonage's servers to communicate with yours.
 
-For your application to be accessible to Nexmo's servers, it must be publicly available on the Internet. A simple way to achieve this during development and testing is to use [ngrok](https://ngrok.com), a service that exposes local servers to the public Internet over secure tunnels. See [this blog post](https://www.nexmo.com/blog/2017/07/04/local-development-nexmo-ngrok-tunnel-dr/) for more details.
+For your application to be accessible to VOnage's servers, it must be publicly available on the Internet. A simple way to achieve this during development and testing is to use [ngrok](https://ngrok.com), a service that exposes local servers to the public Internet over secure tunnels. See [this blog post](https://www.nexmo.com/blog/2017/07/04/local-development-nexmo-ngrok-tunnel-dr/) for more details.
```
Comment (tbedford) [inline _use_cases/en/number-insight-async-tutorial.md:66]: Typo in Vonage.

File: _use_cases/en/trancribe-amazon-api.md [hunk 6]
```diff
@@ -555,14 +555,14 @@ recording...
...
-Getting transcription job: transcript-nexmo-srWr3XOmP.mp3
+vonage-srWr3XOmP.mp3 uploaded to vonage-transcription-audio bucket
+Submitting file https://s3-us-east-1.amazonaws.com/vonage-transcription-audio/vonage-srWr3XOmP.mp3 for transcription...
+Started transcription job transcript-vonage-srWr3XOmP.mp3...
+transcript-vonage-srWr3XOmP.mp3 job successful
+Getting transcription job: transcript-vonage-srWr3XOmP.mp3
 Retrieved transcript
-downloading transcript-nexmo-srWr3XOmP.mp3.json
-Transcript: ./transcripts/transcript-nexmo-srWr3XOmP.mp3.json has been created.
+downloading transcript-vonage-srWr3XOmP.mp3.json
```
Comment (tbedford) [inline _use_cases/en/trancribe-amazon-api.md:564]: Will changing these URLs break anything?

**Review by tbedford — CHANGES_REQUESTED** [review]
Couple of minor fixes.

Should API Developer Portal actually be Vonage API Developer Portal?

File: README.md (hunk 2 above)
Comment (marklewin) [reply README.md:1]: Yes, that definitely sounds better!

File: README.md (hunk 3 above)
Comment (marklewin) [reply README.md:123]: Changed all occurences

File: _extend/azure-speech-to-text.md (hunk 4 above)
Comment (marklewin) [reply _extend/azure-speech-to-text.md]: Oops! Forgot to run spell checker ... good spot.

File: README.md (hunk 3 above)
Comment (marklewin) [reply README.md]: Changed all occurences

File: _use_cases/en/trancribe-amazon-api.md (hunk 6 above)
Comment (marklewin) [reply _use_cases/en/trancribe-amazon-api.md:564]: No, these names are arbitrary and set by the reader during the tutorial.

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (marklewin) [reply .github/CONTRIBUTING.md:1]: You're right - reverted.

**Comment by marklewin** [comment]
> Should API Developer Portal actually be Vonage API Developer Portal?

It definitely sounds better, so let's go with that!

**Review by tbedford — APPROVED** [review]

**Pull request merged**